MathLib/LinAlg/Lis/LisMatrix.h

```diff
@@ -45,6 +45,22 @@ struct SetMatrixSparsity<LisMatrix, SPARSITY_PATTERN>;
 class LisMatrix
 {
 public:
+    /// Matrix type
+    enum class MatrixType : int
```

Comment:
But for the time being the matrix type is not passed on to Lis, so it wouldn't hurt to remove that code.
But I won't insist on that.